tests/Integration/Behaviour/Features/Scenario/Address/customer_address.feature

```diff
@@ -0,0 +1,265 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s address --tags customer-address
+@reset-database-before-feature
+@customer-address
+Feature: Address
+  PrestaShop allows BO users to manage customer addresses
+  As a BO user
+  I should be able to customize customer addresses
+
+  # NOTE: these scenario cannot be run independently you need to run them all in the right order
```

Comment:
Then maybe it should be one big scenario ?

Reply:
It could but it would become a bit hard to read.. Here each scenario is a different use case, the different scenarii are only dependent because of the total number of addresses in database